tango_simlib/tango_sim_generator.py

```diff
@@ -285,7 +286,10 @@ def initialize_dynamic_attributes(self):
                 # device class prior to start-up.
```

Comment:
Suggest you add to these comments to say why the IMAGE type is special and doesn't get added as a static or dynamic attribute.